src/supermarket/app/controllers/api/v1_controller.rb

```diff
@@ -46,7 +46,7 @@ def error(body, status = 400)
   #
   def init_params
     @start = params.fetch(:start, 0).to_i
-    @items = [params.fetch(:items, 10).to_i, 100].min
+    @items = [params.fetch(:items, 10).to_i, item_limit].min
```

Comment:
Opened #1544 to return an error when an item count is requested greater than the item limit.